site/docs/tutorial/ios-app.md

```diff
@@ -198,6 +209,7 @@ ios_application(
     ],
     minimum_os_version = "9.0",
     infoplists = [":UrlGet/UrlGet-Info.plist"],
+    launch_storyboard = "UrlGet/UrlGetViewController.xib",
```

Comment:
Instead I think you'll want to add a dummy storyboard instead of reusing the view controller's XIB

Reply:
I just referred the [BUILD](https://github.com/bazelbuild/examples/blob/master/tutorial/ios-app/BUILD) file, but I didn't intend to add it so I'm going to remove `launch_storyboard` attribute. This is because the addition needs to update `source-only` branch of [bazelbuild/examples](https://github.com/bazelbuild/examples/). Thank you.